packages/services/src/contents/index.ts

```diff
@@ -108,7 +108,7 @@ export namespace Contents {
   /**
    * A contents file type.
    */
-  export type ContentType = 'notebook' | 'file' | 'directory';
+  export type ContentType = 'notebook' | 'file' | 'directory' | 'dirlike';
```

Comment:
I think this is supposed to match the spec here: https://jupyter-notebook.readthedocs.io/en/stable/extending/contents.html

Reply:
You're right, that's potentially problematic since you can pass an override `ContentType` to the contents api.

I fixed this by widening the `IFileType.contentType` property itself. The property isn't used in direct conjunction with the contents api anywhere, so there's no need for it to match.